src/Orleans.Transactions/Hosting/SiloBuilderExtensions.cs

```diff
@@ -8,38 +9,63 @@ namespace Orleans.Transactions
 {
     public static class SiloBuilderExtensions
     {
+        /// <summary>
+        /// Configure cluster to use an in-cluster transaction manager using defaults.
+        /// </summary>
+        public static ISiloHostBuilder UseInClusterTransactionManager(this ISiloHostBuilder builder)
+        {
+            return builder.UseInClusterTransactionManager(options => { });
+        }
+
         /// <summary>
         /// Configure cluster to use an in-cluster transaction manager.
         /// </summary>
-        public static ISiloHostBuilder UseInClusterTransactionManager(this ISiloHostBuilder builder, TransactionsConfiguration config)
+        public static ISiloHostBuilder UseInClusterTransactionManager(this ISiloHostBuilder builder, Action<TransactionsOptions> configureOptions)
```

Comment:
You are missing the overload that takes both `ISiloHostBuilder` and `Action<OptionsBuilder<TransactionsOptions>>`